src/@chakra-ui/gatsby-plugin/components/components.utils.ts

```diff
@@ -0,0 +1,70 @@
+import { SystemStyleObject, theme } from "@chakra-ui/react"
+import { merge } from "lodash"
+
+const {
+  Accordion: accordionDefaultTheme,
+  Avatar: avatarDefaultTheme,
+  Badge: badgeDefaultTheme,
+  Breadcrumb: breadcrumbDefaultTheme,
+  Button: buttonDefaultTheme,
+  Checkbox: checkboxDefaultTheme,
+  CloseButton: closeButtonDefaultTheme,
+  Code: codeDefaultTheme,
+  Divider: dividerDefaultTheme,
+  Drawer: drawerDefaultTheme,
+  Form: formDefaultTheme,
+  FormLabel: formLabelDefaultTheme,
+  Heading: headingDefaultTheme,
+  Input: inputDefaultTheme,
+  Link: linkDefaultTheme,
+  List: listDefaultTheme,
+  Menu: menuDefaultTheme,
+  Modal: modalDefaultTheme,
+  Select: selectDefaultTheme,
+  Spinner: spinnerDefaultTheme,
+  Switch: switchDefaultTheme,
+  Table: tableDefaultTheme,
+  Tabs: tabsDefaultTheme,
+  Tag: tagDefaultTheme,
+} = theme.components
```

Comment:
Import aliases used for consistency, as some of these defaults are exported to custom component configs.

The rest are imported to retain the current UI with all Chakra components used that have themes, and includes themes for components that may be used later as the UI migration continues and other refactoring needs.

It is possible that this list could be trimmed down with any discovery of themes that are not actually needed.